tests/sample/hash_of_map_in_array_map.c

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+// Whenever this sample program changes, bpf2c_tests will fail unless the
+// expected files in tests\bpf2c_tests\expected are updated. The following
+// script can be used to regenerate the expected files:
+//     generate_expected_bpf2c_output.ps1
+//
+// Usage:
+// .\scripts\generate_expected_bpf2c_output.ps1 <build_output_path>
+// Example:
+// .\scripts\generate_expected_bpf2c_output.ps1 .\x64\Debug\
+
+#include "bpf_helpers.h"
+
+#define LB_MAGLEV_MAP_MAX_ENTRIES 65536
```

Comment:
Please change the constant to have some other identifier than maglev.

Reply:
I think we should remove maglev name from the file.

Reply:
Can you clarify the reason not to have 'maglev' string?